skeletorscript.py

```diff
@@ -1256,11 +1256,17 @@ def MakeBOSLineString(turn_or_move, bonename, axisindex, targetposition, speed,
         elif ISDEATH:
             # TODO for death animations:
             # turn values and speeds probably need to be converted to radians
-            outf.write(
-                "-- use call-script DeathAnim(); from Killed()\nDeathAnim() {--  %s \n\tsignal SIG_WALK;\n\tsignal SIG_AIM;\n\tcall-script StopWalking();\n\tturn aimy1 to y-axis <0> speed <120>;\n\tturn aimx1 to x-axis <0> speed <120>;\n" % (
-                    INFOSTRING))
+            outf.write("""
+-- use StartThread(DeathAnim) from Killed()
+local function DeathAnim() -- %s
+\tSignal(SIG_WALK)
+\tSignal(SIG_AIM)
+\tStartThread(StopWalking()
+\tTurn(aimy1, y_axis, 0, %d)

```

Comment:
question: these aimx / aimy parts - they need to be added to the model?

Reply:
The aimx and aimy, as per the documentation, are needed so that animations are 'independent' of aiming pitch and heading. They are not required per se, and can be ommited from model, but are quite useful when animating torsos and arms.

Reply:
doh, I had seen them mentioned but apparently missed the dot point where you explain exactly what/where they go. Thanks!